.github/workflows/main.yml

```diff
@@ -52,7 +52,7 @@ jobs:
           pip install --progress-bar off .[extras]
           if [[ ${{ matrix.torch-version }} == "1.4.0" ]]; then pip install pylint==2.4.4 flake8==3.7.9; fi
           if [[ ${{ matrix.torch-version }} != "1.0.1" ]]; then pip install mypy==0.790; fi
-          pip install pytest
+          pip install pytest six
```

Comment:
I think you should list `six` as a requirement in `requirements.txt` and in `install_requires` in `setup.py` instead of hard-coding it here.